empyrical/stats.py

```diff
@@ -1005,6 +1005,41 @@ def cagr(returns, period=DAILY, annualization=None):
     return ending_value ** (1. / no_years) - 1
 
 
+def value_at_risk(returns, cutoff):
```

Comment:
If this gets passed a 2-d array (which would match the expected input for `conditional_value_at_risk`, then `np.percentile` will calculate the percentile of the flattened array:

```
In [5]: data
Out[5]:
array([[ 0,  1,  2,  3,  4],
       [ 5,  6,  7,  8,  9],
       [10, 11, 12, 13, 14],
       [15, 16, 17, 18, 19],
       [20, 21, 22, 23, 24]])

In [6]: np.percentile(data, 30)
Out[6]: 7.1999999999999993
```

As a user, I would expect `value_at_risk` and `conditional_value_at_risk` to have the same input/output signature.